src/api/libcellml/module/libcellml

```diff
@@ -24,13 +24,15 @@ limitations under the License.
  * \section intro_sec Introduction
  *
  * This is the source code documentation for the libCellML C++ library.
- *
  */
-#include <libcellml/version.h>
-#include <libcellml/model.h>
 #include <libcellml/component.h>
+#include <libcellml/error.h>
 #include <libcellml/import.h>
+#include <libcellml/logger.h>
+#include <libcellml/model.h>
+#include <libcellml/parser.h>
 #include <libcellml/units.h>
 #include <libcellml/variable.h>
+#include <libcellml/version.h>
```

Comment:
Nitpicking: I believe we agreed on using `"` rather than `<`/`>` for in-project headers? (Same in `tests/coverage/coverage.cpp` and `tests/parser/parser.cpp`, although they are tests, so they might actually be fine...?)